draft-ietf-quic-recovery.md

```diff
@@ -1518,20 +1522,20 @@ newly acked_packets from sent_packets.
    OnPacketsAcked(acked_packets):
      for (packet in acked_packets):
        // Remove from bytes_in_flight.
-       bytes_in_flight -= packet.size
+       bytes_in_flight -= packet.sent_bytes
        if (InCongestionRecovery(packet.time_sent)):
          // Do not increase congestion window in recovery period.
-         return
+         continue
        if (IsAppOrFlowControlLimited()):
          // Do not increase congestion_window if application
          // limited or flow control limited.
-         return
+         continue
        if (congestion_window < ssthresh):
          // Slow start.
-         congestion_window += packet.size
-         return
+         congestion_window += packet.sent_bytes
+         continue
```

Comment:
Might be `else`, but not `continue`.

Reply:
But this should not be `return`! I like `else`.